ui/component/repostCreate/view.jsx

```diff
@@ -0,0 +1,294 @@
+      <Card
+        actions={
+          <div>
+            {name && (
+              <>
+                <FormField
+                  label={'Content to repost'}
+                  type="text"
+                  name="repost_url"
+                  value={enteredContentUri}
+                  error={false}
+                  onChange={event => setEnteredContentUri(event.target.value)}
+                />
+                <div className="form-field__help">{`Enter a name or copy and paste a ${
+                  SITE_NAME === 'lbry.tv' ? SITE_NAME : `LBRY or ${SITE_NAME}`
+                } URL`}</div>
+              </>
+            )}
+            <fieldset-section>
+              {(uri || contentUri) && (
+                <ClaimPreview
+                  key={uri || contentUri}
+                  uri={uri || contentUri}
+                  actions={''}
+                  type={'large'}
+                  showNullPlaceholder
+                />
+              )}
+              {!uri && !contentUri && (
+                <ClaimPreview actions={''} type={'large'} placeholder={'loading'} showNullPlaceholder />
+              )}
+            </fieldset-section>
+
+            <React.Fragment>
+              <fieldset-section>
+                <fieldset-group class="fieldset-group--smushed fieldset-group--disabled-prefix">
+                  <fieldset-section>
+                    <label>{__('Repost url')}</label>
+                    <div className="form-field__prefix">{repostUrlName}</div>
+                  </fieldset-section>
+                  <FormField
+                    type="text"
+                    name="repost_name"
+                    value={enteredRepostName}
+                    error={repostNameError}
+                    onChange={event => setEnteredRepostName(event.target.value)}
+                  />
+                </fieldset-group>
+              </fieldset-section>
+              <div className="form-field__help">
+                <I18nMessage
+                  tokens={{
+                    lbry_naming_link: (
+                      <Button button="link" label={__('community name')} href="https://lbry.com/faq/naming" />
+                    ),
+                  }}
+                >
+                  Change this to repost to a different %lbry_naming_link%.
+                </I18nMessage>
+              </div>
+
+              <SelectChannel
+                label={__('Channel to repost on')}
+                // hideAnon
+                hideNew
+                channel={repostChannel}
+                onChannelChange={newChannel => setRepostChannel(newChannel)}
+              />
+
+              <FormField
+                type="number"
+                name="repost_bid"
+                min="0"
+                step="any"
+                placeholder="0.123"
+                className="form-field--price-amount"
+                label={<LbcSymbol postfix={__('Stake')} size={14} />}
```

Comment:
I think this should be `Support` instead of `Stake`

Reply:
It was "Deposit"..